assets/app/views/modals/delete-resource.html

```diff
@@ -4,6 +4,21 @@
       '<strong>{{displayName ? displayName : resourceName}}</strong>'?</h1>
     <p>This<span ng-if="isProject"> will <strong>delete all resources</strong> associated with the project {{displayName ? displayName : resourceName}} and</span> <strong>cannot be undone</strong>.  Make sure this is something you really want to do!</p>
 
+    <div ng-show="typeNameToConfirm">
+      <p>Type the name of the {{typeDisplayName || (kind | humanizeKind)}} to confirm.</p>
```

Comment:
We need typeDisplayName for Deployment vs Replication Controller.

I'll remove the type display name from project since I opened another PR to add the option to the filter.